### ISSUE TITLE
Can not find a model after training - strange

### ISSUE THREAD

**Issue by GraphGrailAi** [microsoft/DialoGPT#38]
Hi, cool repo.
I fine-tune `small_ft.pkl` with the following training parameter in Google Colab:

`!python LSP_train.py --model_name_or_path /content/DialoGPT/models/small --init_checkpoint /content/apex/DialoGPT/small_ft.pkl --train_input_file /content/apex/DialoGPT/data/train_raw_friend.128len.db --eval_input_file ./data/dummy_data.tsv --output_dir /content/DialoGPT/models/output_model --seed 42 --max_seq_length 128 --train_batch_size 16 --gradient_accumulation_steps 8 --eval_batch_size 64 --learning_rate 1e-5 --num_optim_steps 1000 --valid_step 5000 --warmup_steps 4000 --normalize_data true --fp16 false --lr_schedule noam --loss_scale 0.0 --no_token_id true --pbar true`

Trainig finished but when i navigate to `/content/DialoGPT/models/output_model` there is no actual model ! i see only 2 files in generated folder `GPT2.1e-05.2.1gpu.2020-04-04161237`
eval_log and train_log, but no large pkl file.

What i am doing wrong?

![Screenshot_2020-04-04 Google Colaboratory](https://user-images.githubusercontent.com/4690353/78456760-40192c00-76ae-11ea-95d9-75a40dd084f1.png)

**Comment by intersun** [comment]
Maybe you could print some debug information before you dump the model?

After a quick scan of you command I noticed that your num_optim_steps is 1000 but --valid_step is 5000, noticed that condition to save model is
```
if global_step % args.valid_step == 0
```
, so it looks your training haven't finished "one epoch" to save the model yet.

Hope it helps.

**Comment by GraphGrailAi** [comment]
> 
> 
> Maybe you could print some debug information before you dump the model?
> 
> After a quick scan of you command I noticed that your num_optim_steps is 1000 but --valid_step is 5000, noticed that condition to save model is
> 
> ```
> if global_step % args.valid_step == 0
> ```
> 
> , so it looks your training haven't finished "one epoch" to save the model yet.
> 
> Hope it helps.

Yes, closing
